Phraseanet-production-client/src/components/record/recordEditor/index.js

```diff
@@ -640,8 +654,10 @@ const recordEditorService = services => {
 
                         if (field.input_disable) {
                             $editDateArea.prop('disabled', true);
+                            $buttonEmptyField.show();
```

Comment:
The button can be useful even if the field is not "restricted" ; The only reason to hide the button is if the field is empty (do not do if too complicated).